### DISCUSSION TITLE
set cookies is not set in browser cookies

### DISCUSSION THREAD

**Discussion by shahensaifullah** [nuxt/nuxt#17207] [Questions]
![image](https://user-images.githubusercontent.com/44544213/180660883-3f9e7f94-6ea3-48cf-a976-e4a2a007a6d1.png)
`$fetch(`http://127.0.0.1:8000/api/auth/csrf/`)`
why cookies are not set in browser cookies?

**Comment by webfansplz** [comment]
```ts

 const response = await $fetch(url, {
      credentials: 'include',
    })

```
